spec/serializers/current_order_serializer_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Api::CurrentOrderSerializer do
+  let(:distributor) { create(:distributor_enterprise) }
+  let(:oc) { create(:simple_order_cycle) }
+  let(:li) { create(:line_item, variant: create(:variant)) }
```

Comment:
could we take the chance to improve the variables names to make them a bit more communicative? I'd rather name them `order_cycle` and `line_item` respectively.

Reply:
:+1: done.
I have also added andand to the shipping_method_id method to make it work even if there are no shipments in the order.